console/frontend/src/main/frontend/src/app/views/status/status.component.html

```diff
@@ -84,15 +85,21 @@
     <div class="col-lg-12 tabs-container">
       <ul class="nav nav-tabs" role="menu" uib-dropdown-menu="">
         <li [ngClass]="{ active: selectedConfiguration === 'All' }">
-          <a (click)="changeConfiguration('All')">All</a>
+          <a
+            [attr.data-cy-status-select-configuration]="'All'"
+            (click)="changeConfiguration('All')"
+            >All</a
+          >
```

Comment:
I tested with a simple HTML file that HTML allows you to add additional spacing inside the closing tag. These extra spaces were required by ESLint.